cypress/e2e/gui/engage-sphere.spec.cy.js

```diff
@@ -0,0 +1,378 @@
+                cy.get('button').contains('View').click();
+                cy.checkA11y();
+            });
+
+            it('Messenger contact form should have no accessibility issues', () => {
+                //fiquei em dúvida, se há necessidade de ajustar o component Messenger Contact Form ou easter egg
+                cy.getByClassThatStartsWith('Messenger_openCloseButton').click();
+                cy.checkA11y();
+            });
+        });
+
+        context('Dark Mode', () => {
+            beforeEach(() => {
+                cy.getByClassThatStartsWith('ThemeToggle_button').click();
+                cy.get('[data-theme="dark"]').should('exist');
+            });
+
+            it('Customer table should have no accessibility issues in dark mode', () => {
+                cy.checkA11y();
+            });
+
+            it('Customer details and address view should have no accessibility issues in dark mode', () => {
+                cy.get('button').contains('View').click();
+                cy.checkA11y();
+            });
+
+            it('Messenger contact form should have no accessibility issues in dark mode', () => {
+                //fiquei em dúvida, se há necessidade de ajustar o component Messenger Contact Form ou easter egg
+                cy.getByClassThatStartsWith('Messenger_openCloseButton').click();
+                cy.checkA11y();
+            });
+        });
+    });
+
+    context('Empty State', () => {
+        beforeEach(() => {
+            cy.intercept('GET', `${CUSTOMERS_API_URL}?page=1&limit=10&size=All&industry=All`,
+                { fixture: 'empty' },
+            ).as('getEmptyCustomers');
+
+            cy.visit('/');
+            cy.wait('@getEmptyCustomers');
+        });
+
+        it('Displays an empty state illustration and the message “No customers available” when the customer database is empty.', () => {
+            cy.get('svg[title="image of an empty box"]').should('be.visible');
+            cy.contains('span', 'No customers available.').should('be.visible');
+        });
+
+        it('Disables the customer name input field when the customer database is empty.', () => {
+            cy.get('#name').should('be.disabled');
+        });
+    });
+
+    context('Messenger Contact Form', () => {
+        it('Displays a success message after the messenger form is successfully submitted, and hides it automatically after a short delay.', () => {
+            const messengerData = {
+                name: 'Matheus',
+                email: 'matheus@gmail.com',
+                message: 'Hello, I need help with something.',
+            };
+
+            cy.getByClassThatStartsWith('Messenger_openCloseButton').click();
+            cy.get('#messenger-name').type(messengerData.name);
+            cy.get('#email').type(messengerData.email);
+            cy.get('#message').type(messengerData.message);
+            cy.clock();
+            cy.getByClassThatStartsWith('Messenger_sendButton').click();
```

Comment:
Aqui poderia ser algo mais simples, como:

```suggestion
            cy.contains('button', 'Send').click();
```